app/pods/classroom/timeline/library/controller.js

```diff
@@ -27,6 +27,11 @@ export default class ClassroomTimelineLibraryController extends Controller {
       component: 'course-library/bookmarks',
       task: this.fetchBookmarks
 
+    },
+    {
+      name: 'Doubts',
+      component: 'player/player-doubts-list',
```

Comment:
create a new component `course-library/doubts` which wraps `Player::PlayerDoubtsList`
This will keep the pattern consistent and you won't need to rename parameter names to `payload`